daemon/src/maker_inc_connections_actor.rs

```diff
@@ -43,7 +43,7 @@ pub fn new(
         loop {
             tokio::select! {
                 Ok((socket, remote_addr)) = listener.accept() => {
-                    println!("Connected to {}", remote_addr);
+                    tracing::info!(%remote_addr, "Connected to");
```

Comment:
I'm happy to change it back. 
For the other interested parties, this is how it would look like in json format: 

```json
{
  "timestamp": "....",
  "level": "INFO",
  "fields": {
    "message": "Listening on",
    "local_addr": "0.0.0.0:9999"
  }
}
```